### DISCUSSION TITLE
Import does not work as described in the documentation

### DISCUSSION THREAD

**Discussion by SvenBudak** [nolimits4web/swiper#4980] [General]
I am using Vue 2.6.11 and followed these steps: https://swiperjs.com/vue

Unfortunately, the following did not work:
```
// Import Swiper Vue.js components
import { Swiper, SwiperSlide } from 'swiper/vue';

// Import Swiper styles
import 'swiper/css';
```

I had to modify it like this:
```
// Import Swiper Vue.js components
import {Swiper, SwiperSlide} from "swiper/vue/swiper-vue";

// Import Swiper styles
import 'swiper/swiper.scss';
```

but now I get the error:
```
This dependency was not found:

* swiper in ./node_modules/swiper/vue/get-params.js
```

What could be the reason?

**Comment by Charlene-Bx** [comment]
Hello, i've the same issue and after a long time passed on different topic from Stack Overflow I discovered that the current version of vue Swiper only works with vue3... I think it would be important to clarify this in the current documentation. [This topic](https://github.com/nolimits4web/swiper/discussions/4759) is on the same issue.
